thrift/src/main/thrift/atoms/explainer.thrift

```diff
@@ -2,16 +2,59 @@ namespace * contentatom.explainer
 namespace java com.gu.contentatom.thrift.atom.explainer
 #@namespace scala com.gu.contentatom.thrift.atom.explainer
 
+include "../shared.thrift"
+
 enum DisplayType {
   FLAT = 0,
   EXPANDABLE = 1,
   CAROUSEL = 2
 }
 
 struct ExplainerAtom {
```

Comment:
Can we not call these `explainers` please given we already have an 'explainer' atom.